backend/src/src-moderation/src/main/java/org/eclipse/sw360/moderation/ModerationHandler.java

```diff
@@ -316,12 +323,20 @@ public Map<String, Long> getCountByModerationState(User user) throws TException
 
     @Override
     public Map<PaginationData, List<ModerationRequest>> getRequestsByModeratorWithPagination(User user,
-            PaginationData pageData, boolean open) throws TException {
+                                                                                             PaginationData pageData, boolean open) throws TException {
```

Comment:
Please remove extra spaces on lines 326 and 334.

Reply:
Removed, please check.